### ISSUE TITLE
No ecash after update from v0.1.7 beta5 to v0.1.7 beta12

### ISSUE THREAD

**Issue by GitKalle** [minibits-cash/minibits_wallet#73]
After OTA from v0.1.7 beta5 to 0.1.7 beta12 all my amount was 0. Then I delete the internal memory of the APP and try to recover but I get always an error:

```
Recovery failed
Recovery ended with errors
(show errors)

[object Object]
https://mint.minibits.cash/bitcoin
```

I recover my ecash successfully with nutshell.

**Comment by minibits-cash** [comment]
I wrote general steps on how to dig out from broken migration to your related issue.

https://github.com/minibits-cash/minibits_wallet/issues/69#issuecomment-2115218216

Happy you were able to get your sats back. I tested the recovery process and as I can not simulate the broken state after the failed migration, I could not get the error - recovery works fine.

That's why I recommend in such case to do full wallet reinstall (not only factory reset) and recover as you did in another wallet.
Sorry for your troubles - if I could simulate the migration issues, those would have been fixed already.
